perf/micro/immediate-scheduler/operators/elementat.js

```diff
@@ -0,0 +1,20 @@
+var RxOld = require("rx");
+var RxNew = require("../../../../index");
+
+module.exports = function (suite) {
+
+  var oldElementAtWithImmediateScheduler = RxOld.Observable.range(0, 25, RxOld.Scheduler.immediate).elementAt(5);
```

Comment:
As same as test case https://github.com/ReactiveX/RxJS/pull/303, this operator does not accept any schedulers. I do not feel let those kind of test belong under current micro test category (immediate / current thread) is correct categorization, maybe need new category such as 'default', or others?